sample-lnd.conf

```diff
@@ -515,6 +515,8 @@ bitcoin.node=btcd
 ; bitcoin.dnsseed=seed1.test.lightning
 ; bitcoin.dnsseed=seed2.test.lightning,soa.seed2.test.lightning
 
+; Whether or not to wait for a channel to confirm on-chain before making it operational.
+; bitcoin.skip-channel-confirmation=false
```

Comment:
I think the behavior of this is also weird in that, if you set this argument, then you automatically accept _everything_ as a zero conf channel, rather than only accepting them in certain situations.